packages/manager/modules/support/src/tickets/new-ticket/new-ticket.html

```diff
@@ -19,38 +21,37 @@
     <oui-spinner data-size="l"></oui-spinner>
 </div>
 
-<oui-message data-type="success"
-             data-ng-if="$ctrl.step === 'created'">
-    <span data-translate="ovhManagerSupport_new_creation_success"
-          data-translate-values="{ state: 'support.tickets.ticket({ id: $ctrl.ticketId })', id: $ctrl.ticketNumber }"
-          data-translate-compile>
-    </span>
-    <span data-translate="ovhManagerSupport_new_creation_success_back"
-          data-translate-values="{ state: 'support.tickets' }"
-          data-translate-compile>
-    </span>
-</oui-message>
-
 <div data-ng-if="$ctrl.step === 'created'">
+    <oui-message data-type="success">
+        <span data-translate="ovhManagerSupport_new_creation_success"
+                data-translate-values="{ state: 'support.tickets.ticket({ id: $ctrl.ticketId })', id: $ctrl.ticketNumber }"
+                data-translate-compile>
+        </span>
+        <span data-translate="ovhManagerSupport_new_creation_success_back"
+                data-translate-values="{ state: 'support.tickets' }"
+                data-translate-compile>
+        </span>
+    </oui-message>
+
     <p data-translate="ovhManagerSupport_new_creation_tip_1"></p>
     <p data-ng-if="$ctrl.guideURL">
         <span data-translate="ovhManagerSupport_new_creation_tip_guides"></span>
         <a data-ng-href="{{ $ctrl.guideURL }}"
-           target="_blank"
-           data-ng-bind="$ctrl.guideURL">
+            target="_blank"
```

Comment:
No external icon nor `rel=noopener` ?

Reply:
I haven't touched this,  I think you are looking at whitespace changes 😄 

I'll fix this though !

Reply:
Done